lib/march_hare/session.rb

```diff
@@ -103,14 +103,16 @@ def self.connect(options = {})
             kmf.init(ks, pwd)
 
             if options[:trust_manager]
-              ctx.init(kmf.get_key_managers, [options[:trust_manager]], nil)
+              tm_a = options[:trust_manager].kind_of?(Array) ? options[:trust_manager] : [options[:trust_manager]]
```

Comment:
There's `Kernel#Array` that does this.

Reply:
Ah, so Array(options[:trust_manager])?  Much simpler.  I'll fix.

Reply:
Done